src/analytics/redux/tracked_events.js

```diff
@@ -111,20 +221,35 @@ function handleBlockMovedByPosition( clientIds, toIndex ) {
 
 export const trackedEvents = {
 	'core/block-editor': {
-		insertBlock( blocks ) {
-			trackBlocksHandler(
+		insertBlock( blocks, index, rootClientId, updateSelection, meta ) {
+			trackBlockInsertion(
 				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
+				index,
+				rootClientId,
+				updateSelection,
+				undefined,
+				meta
 			);
 		},
-		insertBlocks( blocks ) {
-			trackBlocksHandler(
+		insertBlocks(
+			blocks,
+			index,
+			rootClientId,
+			updateSelection,
+			initialPosition,
+			meta
+		) {
+			trackBlockInsertion(
 				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
+				index,
+				rootClientId,
+				updateSelection,
+				initialPosition,
+				meta
```

Comment:
Similar to [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976#discussion_r1268329007), we could remove the unused arguments.

Reply:
Updated it in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/94df0273bad2f0dc1a9422b4a590e5ecb7b3beb5 thanks!